lib/pages/login.dart

```diff
@@ -258,56 +401,44 @@ class _LoginScreenState extends State<LoginScreen> {
                     begin: Alignment.topCenter,
                     end: Alignment.bottomCenter,
                     colors: [
+                      Color(0xFF91d18b),
                       Color(0xFF519872),
-                      Color(0xFF158467),
                       Color(0xFF206a5d),
-                      Color(0xFF184d47),
+                      Color(0xFF004a2f),
                     ],
                     stops: [0.1, 0.4, 0.7, 0.9],
                   ),
                 ),
               ),
-              Container(
-                padding: EdgeInsets.all(50.0),
-                child: Image.asset(
-                  'assets/logos/ClipartKey_56184.png',
-                  width: 70,
-                  height: 70,
-                ),
-              ),
               Container(
                 height: double.infinity,
                 child: SingleChildScrollView(
                   physics: AlwaysScrollableScrollPhysics(),
                   padding: EdgeInsets.symmetric(
                     horizontal: 40.0,
-                    vertical: 100.0,
+                    vertical: 60.0,
                   ),
-                  child: Column(
-                    mainAxisAlignment: MainAxisAlignment.center,
-                    children: <Widget>[
-                      Text(
-                        'Sign In',
-                        style: TextStyle(
-                          color: Colors.white,
-                          fontFamily: 'OpenSans',
-                          fontSize: 40.0,
-                          fontWeight: FontWeight.bold,
+                  child: Form(
+                    key: _formKey,
+                    child: Column(
+                      mainAxisAlignment: MainAxisAlignment.center,
+                      children: <Widget>[
+                        _buildLogo(),
+                        SizedBox(height: 20.0),
+                        _buildEmailTF(),
+                        SizedBox(
+                          height: 20.0,
                         ),
-                      ),
-                      SizedBox(height: 30.0),
-                      _buildEmailTF(),
-                      SizedBox(
-                        height: 30.0,
-                      ),
-                      _buildPasswordTF(),
-                      _buildForgotPasswordBtn(),
-                      _buildRememberMeCheckbox(),
-                      _buildLoginBtn(),
-                      _buildSignInWithText(),
-                      _buildSocialBtnRow(),
-                      _buildSignupBtn(),
-                    ],
+                        _buildPasswordTF(),
+                        _buildForgotPasswordBtn(),
+                        _buildRememberMeCheckbox(),
+                        _buildLoginBtn(),
+                        _buildSignInWithText(),
+                        _buildSocialBtnRow(),
```

Comment:
Make all the buttons have a similar width.
![image](https://user-images.githubusercontent.com/7119799/95019003-ef6c9680-0630-11eb-8abe-42471c64b54a.png)

Reply:
Done.